geode-core/src/test/java/org/apache/geode/internal/cache/xmlcache/CacheXmlParserJUnitTest.java

```diff
@@ -111,10 +113,31 @@ public void testCacheXmlParserWithSimplePool() {
     InternalDistributedSystem system =
         InternalDistributedSystem.newInstanceForTesting(dm, nonDefault);
     when(dm.getSystem()).thenReturn(system);
-    InternalDistributedSystem.connect(nonDefault);
 
-    CacheXmlParser.parse(getClass()
-        .getResourceAsStream("CacheXmlParserJUnitTest.testSimpleClientCacheXml.cache.xml"));
+    Cache cache = new CacheFactory()
+        .set("cache-xml-file", "CacheXmlParserJUnitTest.testSimpleClientCacheXml.cache.xml")
+        .create(InternalDistributedSystem.connect(nonDefault));
+    cache.close();
+  }
+
+  /**
+   * Test that {@link CacheXmlParser} can parse the test cache.xml file, using the Apache Xerces XML
+   * parser.
+   * 
+   * @since Geode 1.3
+   */
+  @Test
+  public void testCacheXmlParserWithSimplePoolXerces() {
+    String prevParserFactory = System.setProperty("javax.xml.parsers.SAXParserFactory",
+        "org.apache.xerces.jaxp.SAXParserFactoryImpl");
+
+    testCacheXmlParserWithSimplePool();
+
+    if (prevParserFactory != null) {
```

Comment:
Thanks Jared! Will find time to make the changes and get feedback on the mailing list.